.eslintrc

```diff
@@ -1,6 +1,34 @@
 {
-  "extends": "react-app",
+  "extends": "amo",
```

Comment:
There's also a `plugin:amo/recommended`. Not sure if that's needed for this repo.

Reply:
Thanks, I've added that now.